packages/zoe/test/unitTests/contracts/test-priceAggregator.js

```diff
@@ -67,8 +67,9 @@ test.before(
       'b1-aggregator',
     );
 
-    const link = makeIssuerKit('$LINK', AssetKind.NAT);
-    const usd = makeIssuerKit('$USD', AssetKind.NAT);
+    const link = makeIssuerKit('$ATOM', AssetKind.NAT);
+    const { brand: atomBrand } = makeIssuerKit('$ATOM', AssetKind.NAT);
+    const { brand: usdBrand } = makeIssuerKit('$USD', AssetKind.NAT);
```

Comment:
`AssetKind.NAT` is the default, yes? Maybe get rid of it here while we're at it?